js/loadThePilesOfCards.js

```diff
@@ -0,0 +1,105 @@
+function feelTheField() {
+	
+}
+function shuffle(a) {
+	var j, x, i;
+	for (i = a.length; i; i -= 1) {
+		j = Math.floor(Math.random() * i);
+		x = a[i - 1];
+		a[i - 1] = a[j];
+		a[j] = x;
+	}
+}
```

Comment:
And I hope that now you know that you can shuffle your array in more simple way using sort)))